src/utils/schema.ts

```diff
@@ -28,3 +32,47 @@ export function parseSchemaResult(schema: z.ZodTypeAny, v: unknown) {
   }
   return schemaResult;
 }
+
+export async function validateAnt({
+  processId,
+  ao = connect(),
+}: {
+  processId: string;
+  ao: AoClient;
+}): Promise<Record<AoANTHandler, { valid: boolean; error?: string }>> {
+  const antHandlers = await getAntHandlers({ processId, ao });
+  const defaultValidations = Object.fromEntries(
+    AntHandlerNames.map((handler) => [
+      handler,
+      async (_: { processId: string; ao: AoClient }) => {
+        if (antHandlers.includes(handler)) return true;
+        throw new Error(`Handler ${handler} not found`);
+      },
+    ]),
```

Comment:
this could be provided as a function to the utility with this as a default, allowing flexibility for how clients want to determine an ANT valid. but fine for now, maybe just add a TODO.